plugins/Marketplace/lang/en.json

```diff
@@ -8,6 +8,9 @@
         "AllowedUploadFormats": "You may upload a plugin or theme in .zip format via this page.",
         "Authors": "Authors",
         "Browse": "Browse",
+        "TryFreeTrialTitle": "Try 30 days for free, then",
```

Comment:
This string seems quite hard to translate without any context (as the grammar might depend on what "then" is)